india_compliance/gst_india/overrides/address.py

```diff
@@ -9,6 +9,24 @@
     validate_pincode,
 )
 
+def set_gstin_and_gstCategory(doc, method=None):
+    links = doc.links
+
+    for link in links:
+        doctype = link.link_doctype
+        docname = link.link_name
+
+        if doctype not in ['Customer','Supplier']:
+            continue
+
+        gstin = doc.gstin
+        if gstin:
+            frappe.db.set_value(doctype,docname,"gstin",gstin)
+
+        gst_category = doc.gst_category
+        if gst_category:
+            frappe.db.set_value(doctype, docname, "gst_category", gst_category)
```

Comment:
only set if gstin exists and gst_category in party (customer/supplier) is unregistered. 
Alert => GSTIN updated in Party (optional)